eks-worker-al2.json

```diff
@@ -13,10 +14,10 @@
     "aws_session_token": "{{env `AWS_SESSION_TOKEN`}}",
     "binary_bucket_name": "amazon-eks",
     "binary_bucket_region": "us-west-2",
-    "cni_plugin_version": "v0.8.6",
-    "containerd_version": "1.6.6-1.amzn2.0.2",
+    "cni_plugin_version": null,
+    "containerd_version": null,
```

Comment:
What if these are empty strings? It will be confusing to customers if this has to be `null` and breaks if they include empty strings. Same with the below.

Reply:
Not sure I'm following; if a customer does `make docker_version=""`, that won't work (same as today).